downgrade

```diff
@@ -484,5 +468,22 @@ if ((!LIB)); then
 
   export TEXTDOMAIN=downgrade TEXTDOMAINDIR
 
+  # check for help CLI option
+  for arg; do
+    if [[ "$arg" =~ ^-h$|^--help$ ]]; then
+      usage
+      exit 1
+    fi
+  done
+
+  # check to ensure downgrade running as root
+  if (($(id -u))); then
```

Comment:
```suggestion
  if ((UID)); then
```

Avoids spawning another process, and just a bit aesthetically nicer, IMO.

Reply:
Makes sense. Added all of the suggestions and cleaned the commit history.

Reply:
PS: Seems like this is a new GitHub feature that every new commit requires a new approval. Either that or I am somehow dismissing your reviews, which I don't know how haha.

In any case, I willl re-request your approval once I am done with the last step ie. locales.

Reply:
That's my fault,

![](https://files.pbrisbin.com/screenshots/screenshot.875542.png)

I must not have hits **Save** when I un-checked this earlier, it was checked.

I made sure to hit Save now, so we should be good going forward. If not, there's a bug in GitHub here somewhere.